modules/ROOT/pages/setup.adoc

```diff
@@ -0,0 +1,114 @@
+= Anypoint Partner Manager Setup
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+You should be familiar with the following concepts before setting up Partner Manager:
+
+* EDI message formats and schemas.
+* Dataweave transformations.
+* Digital Certificates (such as X.509) for the AS2 transport protocol.
+* Your own network of endpoints for the listening to and sending of messages.
+* Troubleshooting in the Anypoint platform.
+
+== Minimum Requirements
+
+Verify that you have the following software and other requirements.
+
+* Mule runtime engine 4.1.1 or higher.
+
+**Note**
+
+Make sure you have the needed Mule runtime Core/Vcore capacity to handle the B2B transaction load. If you are unsure about the capacity you need to handle B2B transactions, please contact your MuleSoft account representative for assistance.
+
+* Storage for your B2B message payloads. 
+
+* Anypoint Studio 7.1 or higher for creating DataWeave maps.
+
+* License for the following optional Connectors:
+
+** AS2 4.0.2 or higher. 
+** X12 2.0.1 or higher. 
+
+* Partner Manager Requirements:
+** Anypoint Platform user account with Anypoint Partner Manager access.
+** Appropriate roles allocated for users to access Anypoint Partner Manager. Roles are allocated via the Access Management section in Anypoint Platform.
+
+**Note**
```

Comment:
This shouldn't be a note, it should probably be the first prerequisite: Check that you can access Anypoint Partner Manager by (tell them where they'll see evidence of it being accessible). then say "If you are unable..."